apps/climatemappedafrica/src/lib/data/blockify/hero.js

```diff
@@ -0,0 +1,20 @@
+import {
+  fetchProfile,
+  fetchProfileGeography,
+} from "@/climatemappedafrica/lib/hurumap";
+
+export default async function hero(block) {
+  const { geometries } = await fetchProfileGeography("af");
```

Comment:
Minor but `"af"` can be set in the CMS, no?

Reply:
@kilemensi Having this in the CMS opens a can of worms, which may not be minor. For instance:

 - What are the options we will allow users to choose from? countries or continents.
 - If continents, then `af` is the only option and we have the centre co-ordinates.
 - If countries, then we also need to set center co-ordinates so that map shows accurately. Can we guarantee that all regions in any country are implemented?
 
 This is only the hero page and would be best to just default to `af` then `countries` become clickable IMO.

Reply:
My understanding is we only need three things @koechkevin:

1. `geography`: We can pull this from HURUmap API and show it as a `Select` field. This way, this information only needs to be entered once (in the backend admin).
2. `centre`: We can use `Point` field (or two `Number` fields) for `lat` and `long`.

>Can we guarantee that all regions in any country are implemented?
>
Isn't the data team handling the validity/correctness of the data?

---

My thinking is a single `Hero` component, should work in any HURUmap based app and hence it should have preferably zero hard-coded props/settings; backend should drive what is shown in it.